packages/permission-controller/CHANGELOG.md

```diff
@@ -7,6 +7,27 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [9.0.0]
+
+### Added
+
+- **BREAKING**: Add ESM build ([#3998](https://github.com/MetaMask/core/pull/3998))
```

Comment:
Nit: I noticed that the colon is sometimes bold and sometimes not. Not a big deal though. 😅

Reply:
Ah, we still need to solve https://github.com/MetaMask/auto-changelog/issues/109.